book/website/reproducible-research/overview/overview-resources.md

```diff
@@ -21,6 +21,30 @@ If you want to start learning hands-on practices, we recommend reading the {ref}
 (rr-overview-resources-addmaterial)=
 ## Additional Material
 
+(rr-overview-resources-commandline)=
+### Using the command line
+
+While most of what this book is about do not require the use of the command line, it often makes things actually easier. 
+Even when your visual abilities allows you to use efficiently your mouse, writing commands in the terminal/powershell/command is not that difficult.
```

Comment:
Suggest we change "not that difficult" to something that explains why its a good idea!